source/_components/climate.opentherm_gw.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
-title: "OpenTherm Gateway"
+title: "OpenTherm Gateway Climate"
 description: "Control your OpenTherm Gateway from Home Assistant."
-date: 2018-08-29 16:23
+date: 2018-10-09 11:23
```

Comment:
Please do not update these dates.
The date indicates the initial introduction of the file.